tsl/test/sql/bgw_job_ddl.sql

```diff
@@ -0,0 +1,74 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+-- Test for DDL-like functionality
+
+\c :TEST_DBNAME :ROLE_SUPERUSER
+CREATE OR REPLACE FUNCTION insert_job(
+       application_name NAME,
+       job_type NAME,
+       schedule_interval INTERVAL,
+       max_runtime INTERVAL,
+       retry_period INTERVAL,
+       owner regrole DEFAULT CURRENT_ROLE::regrole,
+       scheduled BOOL DEFAULT true,
+       fixed_schedule BOOL DEFAULT false
+) RETURNS INT LANGUAGE SQL SECURITY DEFINER AS
+$$
+  INSERT INTO _timescaledb_config.bgw_job(application_name,schedule_interval,max_runtime,max_retries,
+  retry_period,proc_name,proc_schema,owner,scheduled,fixed_schedule)
+  VALUES($1,$3,$4,5,$5,$2,'public',$6,$7,$8) RETURNING id;
+$$;
```

Comment:
Since we're using it in other suite tests as well what about move it to `utils/testsupport.sql`?

Reply:
> Since we're using it in other suite tests as well what about move it to `utils/testsupport.sql`?

It makes sense but since this is just copied from the other BGW tests it would be sensible to take it as a separate refactoring PR and fix the other uses as well.